setup.cfg

```diff
@@ -0,0 +1,47 @@
+[flake8]
+
+# flake8 code - explanation - extension requirement
+# =================================================
+# B,B9 - Bugbear - flake8-bugbear
+# BLK - Black - flake8-black
+# C - complexity - n/a
+# D - docstrings - flake8-docstrings
+# DAR - darglint (checks docstrings) - darglint
+# E - pycodestyle errors - n/a
+# F - pyflakes - n/a
+# I - Import order - flake8-import-order
+# N - PEP8 naming - pep8-naming
+# W - pycodestyle warnings - n/a
+select = B,B9,BLK,C,D,DAR,E,F,I,N,W
+
+# E203 - whitespace before ':'
+# E501 - line length (dealt with by bugbear)
+# W503 - linebreak before binary op
+# B305 - `.next()` is not a thing - metaflow false positive
+# D104 - missing docstring in public package
+# D100 - missing docstring in public module
+ignore = E203,E501,W503,B305,D100,D104
+
+max-complexity = 10
+max-line-length = 80
+application-import-names = metaflow_custom,tests
+import-order-style = smarkets
+docstring-convention = google
+exclude=setup.py
+per-file-ignores =
+    tests/*:D
+
+
+[darglint] # Configuration for the darglint documentation linter
+
+# Don't pester about parameters for a one-line docstring
+strictness=short
+docstring_style=google
+
+[tool:pytest]
+markers = aws: AWS credentials required
+addopts = -vv -m "not aws"
+env =
+    METAFLOW_PROFILE=ds-cookiecutter
```

Comment:
Is this hard-coded to you local setup?

Reply:
Oops, good catch

Reply:
ok cool, I needed this to run the tests on #7